aiographite/aiographitesend.py

```diff
@@ -0,0 +1,447 @@
+import asyncio
+from graphite_escaping import metrics_name_to_graphite, metrics_name_from_graphite
+import pickle
+import struct
+import socket
+import os
+import time
+
+
+DEFAULT_GRAPHITE_PLAINTEXT_PORT = 2003
+DEFAULT_GRAPHITE_PICKLE_PORT = 2004
+SUPPORT_PROTOCOLS = ["plaintext", "pickle"]
+
+
+class AioGraphiteSendException(Exception):
+    pass
+
+
+class AsyncioGraphiteSendService(object):
+
+	def __init__(self, graphite_server, graphite_port, protocol = "pickle"):
+		"""
+			Must call __init__ before use
+		"""
+		# Graphite Server Address
```

Comment:
you probably don't need this comment, since the code is self-explanatory.

I'm not against commenting code: I think it's great. But I also think that readable code ensures we don't need comments, a lot of the time.